### DISCUSSION TITLE
View and edit buttons do not work with generated pages

### DISCUSSION THREAD

**Discussion by francois-rozet** [pradyunsg/furo#808] [Potential Issue] [answered]
For automatically generated `.rst` files, the view and edit buttons lead to 404 GitHub pages. I would expect these buttons to not show up for files that do not exist in the repository.

Example: https://inox.readthedocs.io/en/stable/api/inox.nn.module.html

See also https://github.com/pydata/pydata-sphinx-theme/discussions/1266, https://github.com/pydata/pydata-sphinx-theme/pull/1246

**Comment by pradyunsg** [comment]
Ah autosummary.

**Comment by pradyunsg** [comment, marked as answer]
I'm gonna say I'll tackle this as a part of #620! Thanks for reporting this @francois-rozet!
